internal/storage/bucket_handle.go

```diff
@@ -300,6 +300,12 @@ func (bh *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsReq
 		IncludeFoldersAsPrefixes: req.IncludeFoldersAsPrefixes,
 		//MaxResults: , (Field not present in storage.Query of Go Storage Library but present in ListObjectsQuery in Jacobsa code.)
 	}
+	err = query.SetAttrSelection([]string{"Name", "Size", "Generation", "Metageneration", "Updated", "Metadata"})
```

Comment:
what about unit tests in bucket_handle_test.go?

Reply:
Unit tests in bucket_handle_test.go only assert on object names and do not reference attributes other than these attributes which are fetched. So , the tests do not fail and no additional tests is required.

Reply:
If fake-gcs-server is handling setAttrSelection field, then please add testcases for checking if other properties are not returned.

Reply:
fake-gcs-server is not handling setAttrSelection field.It is returning all attributes when listing .